anima-jws-voucher.mkd

```diff
@@ -55,95 +55,101 @@ informative:
   RFC8949:
   RFC8792:
   RFC8812:
-  onpath:
+  ONPATH:
     target: "https://mailarchive.ietf.org/arch/msg/saag/m1r9uo4xYznOcf85Eyk0Rhut598/"
     title: "can an on-path attacker drop traffic?"
     org: IETF
 
 --- abstract
 
-{{RFC8366}} defines a digital artifact called voucher as a YANG-defined JSON
-document that is signed using a Cryptographic Message Syntax (CMS) structure.
-This document introduces a variant of the voucher artifact in which CMS is
-replaced by the JSON Object Signing and Encryption (JOSE) mechanism described in {{RFC7515}} to support deployments in which JOSE is preferred over CMS.
+RFC8366 defines a digital artifact called voucher as a YANG-defined JSON document that is signed using a Cryptographic Message Syntax (CMS) structure.
+This document introduces a variant of the voucher artifact in which CMS is replaced by the JSON Object Signing and Encryption (JOSE) mechanism described in RFC7515 to support deployments in which JOSE is preferred over CMS.
 
 In addition to explaining how the format is created, the "application/voucher-jws+json" media type is registered and examples are provided.
 
 --- middle
 
 # Introduction
 
-"A Voucher Artifact for Bootstrapping Protocols" {{RFC8366}} defines a YANG-based data structure used in "Bootstrapping Remote Secure Key Infrastructure" {{BRSKI}} and
+"A Voucher Artifact for Bootstrapping Protocols" {{I-D.draft-ietf-anima-rfc8366bis}} defines a YANG-based data structure used in "Bootstrapping Remote Secure Key Infrastructure" {{BRSKI}} and
```

Comment:
My understanding is that this draft is part of the 8366bis cluster and must refer to the -bis version.